### PR TITLE
chore: add maven-zip-build/ta to main kustomization.yaml

### PR DESCRIPTION
**Pull request by ligangty**
This is a missing part of maven-zip-build pipelines. They should be added to main kustomization.yaml to trigger the bundle build. Just added in this PR.

### CODE REVIEW

File: pipelines/maven-zip-build-oci-ta/README.md [hunk 1]
```diff
@@ -1,165 +1,203 @@
 # "maven-zip-build-oci-ta pipeline"
-
```
Comment (MartinBasti) [inline pipelines/maven-zip-build-oci-ta/README.md:2]: Commit message describes need of adding piplienes into main kustomization, why is this doc change required?

Please split README changes into separate commit and describe why change is done

Comment (ligangty) [reply pipelines/maven-zip-build-oci-ta/README.md:2]: The README failed the CI checks so I added it here. I'll split them to another PR.

Comment (ligangty) [reply pipelines/maven-zip-build-oci-ta/README.md]: The README fix is put in this PR: https://github.com/konflux-ci/build-definitions/pull/1692
After its merging I'll do rebase of this PR to it.

Comment (ligangty) [reply pipelines/maven-zip-build-oci-ta/README.md]: Rebase done.

**Comment by chmeliik** [comment]
created the missing repo
/retest

File: pipelines/maven-zip-build-oci-ta/README.md (hunk 1 above)
Comment (MartinBasti) [reply pipelines/maven-zip-build-oci-ta/README.md]: thank you

**Review by MartinBasti — APPROVED** [review]

**Comment by ligangty** [comment]
Seems the CI failed because of a e2e-tests hanging issue. How to fix this?

**Comment by ligangty** [comment]
Rebased and trigger the CI again. Let's see if it can pass this time.

**Review by tkdchen — APPROVED** [review]
